crates/re_viewer_context/src/space_view/named_system.rs

```diff
@@ -22,8 +22,13 @@ pub type PerSystemEntities = BTreeMap<ViewSystemName, BTreeSet<EntityPath>>;
 ///
 /// Required to be implemented for registration.
 pub trait NamedViewSystem {
+    /// Unique name for this system.
+    const NAME: &'static str;
```

Comment:
It's possible, but I made a conscious decision against it for `SpaceViewClass::NAME`, as those are used in other context than checking against existing class names—at the cost of incurring an `.into()` to user code when you do. I did the same for `NamedViewSystem` for consistency, but the argument is weaker there.

Either way, no strong feeling. Happy to make the change for either or both traits, now that I master my IDE's regex-based search and replace :)

Reply:
Scrape that. Can't create `XXXName` in `const` context, so `&str` it is.

Reply:
I only now realise that `NamedViewSystem::name()` doesn't take `&self` (contrary to `SpaceViewClass`), so that change isn't really useful in this case.